app/js/actions/artifacts.js

```diff
@@ -7,11 +14,78 @@ export const expectingArtifact = () => ({
     type: 'EXPECTING_ARTIFACT'
 });
 
-export const deleteArtifact = (uuid) => ({
+export const hiddenDeleteArtifact = (uuid) => ({
     type: 'DELETE_ARTIFACT',
     uuid
 });
 
-export const refreshArtifacts = () => ({
-    type: 'REFRESH_ARTIFACTS'
-});
+export const deleteArtifact = (uuid) => {
+    return (dispatch, getState) => {
+        const { artifacts: { artifacts },
```

Comment:
Oops, another leftover from the first attempt at input artifacts where the artifacts state was split into all of them and their key/value setup that is used in the job state now.